scania-digital-design-system/tegel
# test(slider): added unit tests

**Describe pull-request**  
Adding unit tests for Slider component

**Solving issue**  
Fixes: [CDEP-3011](https://tegel.atlassian.net/browse/CDEP-3011)

**How to test**  
1. Check [unit test spreadsheet for Slider component](https://scaniaazureservices-my.sharepoint.com/:x:/r/personal/tatyana_bumazhkina_scania_com/Documents/Unit%20test%20scenarios.xlsx?d=wca48bf6d92cf4e8c8f456337e2002ec4&csf=1&web=1&e=DarBc4&nav=MTVfezI4OTBDQ0JGLUY2NzgtNDg4OC1BNDY4LUM2QjYxRkY0MzlEQn0).
2. Make sure unit tests in PR do the checks written in the form above 
3. Run code on your local to see that tests pass




[CDEP-3011]: https://tegel.atlassian.net/browse/CDEP-3011?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
